codec-http2/src/main/java/io/netty/handler/codec/http2/CompressorHttp2ConnectionEncoder.java

```diff
@@ -258,7 +274,7 @@ protected EmbeddedChannel newContentCompressor(ChannelHandlerContext ctx, CharSe
         if (DEFLATE.contentEqualsIgnoreCase(contentEncoding) || X_DEFLATE.contentEqualsIgnoreCase(contentEncoding)) {
             return newCompressionChannel(ctx, ZlibWrapper.ZLIB);
         }
-        if (brotliOptions != null && BR.contentEqualsIgnoreCase(contentEncoding)) {
+        if (Brotli.isAvailable() && brotliOptions != null && BR.contentEqualsIgnoreCase(contentEncoding)) {
```

Comment:
Similarly here: Graal cannot native exclude this branch from compilation because it doesn't know at runtime about `brotlyOptions` (a bit weird TBH) -> adding the static check to allow it to work